pages/docs/reference/basic-syntax.md

```diff
@@ -98,7 +98,7 @@ See [Functions](functions.html).
 
 ## Defining variables
 
-Assign-once (read-only) local variable:
+Immutable (assign-once, read-only) local values are defined using the keyword `val`:
```

Comment:
@ilya-g How about:
> Immutable (read-only) local variables are defined using the keyword `val`. They can be assigned a value only once.

I actually prefer your wording, but variables defined by `var` are described as mutable, so we should either be consistent in this heading, or update the following heading.
This edit is a small step that makes the meanings clearer. In the longer run, I'd much prefer to drop jargon like (im)mutable, so that the documentation is accessible to users coming to Kotlin as a first language, but that's a much larger undertaking.